asllib/aslspec/render.ml

```diff
@@ -51,17 +51,25 @@ module Make (S : SPEC_VALUE) = struct
   (** [vars_of_type_term term] returns the list of term-naming variables that
       occur at any depth inside [term]. *)
   let rec vars_of_type_term term =
+    let option_to_list = function Some x -> [ x ] | None -> [] in
```

Comment:
nit: perhaps this helper could handle the pair of name and term, to avoid duplication below?